### ISSUE TITLE
pytestmark isn't working for fixtures

### ISSUE THREAD

**Issue by fruch** [eugeniy/pytest-tornado#10]
using fixtures pytestmark is casuing a test to be skipped:

``` python
    @pytest.fixture(scope="session")
    def preparations():
        pass

    @pytest.fixture
    def app():
        return application

    pytestmark = pytest.mark.usefixtures("preparations")

    @pytest.mark.gen_test
    def test_hello_world(http_client, base_url):
         pass
```

while using is as decorator, it does work:

``` python
    @pytest.mark.usefixtures("preparations")
    @pytest.mark.gen_test
    def test_hello_world(http_client, base_url):
         pass
```

(Didn't had time to actually debug it and check what's the difference, my guess something in the test collection hooks in pytest-tornado)

**Comment by eugeniy** [comment]
I think you're right, it seems to be related to collection of hooks.

Recent py.test adds [hookwrapper] (http://pytest.readthedocs.org/en/latest/plugins.html#hookwrapper-executing-around-other-hooks) which changes how hooks are being invoked. Your example works without any code change with 2.7.0.

**Comment by fruch** [comment]
Thanks... For the information , I would add it to the docs as warning, but
I'm away from my laptop.

**Comment by vidartf** [comment]
As far as I can tell, this was resolved for pytest >= 2.7 (see 8fb97711dd84512a8a654de3dca2bee24689a2a7). Closing for now, but feel free to comment about any remaining problems, and I will reopen.
